requirements.txt

```diff
@@ -55,3 +55,5 @@ typing-extensions==4.12.2
 tzdata==2025.1
 urllib3==2.3.0
 wcwidth==0.2.13
+pynvml>=12.0.0
```

Comment:
these requirements should probably just go in the build requirements for our provider, not in the global requirements for Llama Stack